dev/PipsControl/PipsControl.idl

```diff
@@ -0,0 +1,52 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+runtimeclass PipsControlSelectedIndexChangedEventArgs
+{
+    Int32 NewPageIndex{get; };
+    Int32 PreviousPageIndex{get; };
+};
+
+[WUXC_VERSION_PREVIEW]
+[webhosthidden]
+[MUX_PROPERTY_NEEDS_DP_FIELD]
+unsealed runtimeclass PipsControlTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    PipsControlTemplateSettings();
+
+    Windows.Foundation.Collections.IVector<Object> VerticalPipsItems { get; };
```

Comment:
nit: should probably be just Items. this will be odd if we added an orientation property